docs/patterns/clustered-listening-tentacles.md

```diff
@@ -0,0 +1,144 @@
+
+:::warning
+**Shared Storage Considerations**
+It is not possible to store the **tentacle.config** file in shared storage because the Tentacle.Certificate component gets encrypted using a node's machine-specific key. If you attempt to store the tentacle.config file in shared storage, you will encounter this error: `Key not vaild for use in specified state` upon switching to a new active node. This occurs because the new active node is not able to decrypt the private key resulting in the Tentacle service failing to start.
+:::
+
+To avoid any issues with handling private keys you must specify an alternate location for the **tentacle.config** file upon installation of the Tentacle service. Follow these instructions to install Octopus Tentacle in a failover cluster using shared storage.
+
+## Requirements {#ClusteringTentacles-Requirements}
+
+This guide assumes you already have the following setup:
+
+- An Active Directory Domain and a local DNS server.
+- An Octopus Server (this does not need to be joined to the domain).
+- A two node active/passive Windows cluster where each node is joined to the domain.
+- A local IP Address available for the cluster.
+- Shared Storage configured for the cluster (in this example we are using E:\)
+
+## Installation {#ClusteringTentacles-Installation}
+
+:::warning
+**Installing Tentacles with Shared storage**
+This guide implements shared storage using an iSCSI target with Multipath IO confgured on the Tentacle servers, in this scenario it is best to avoid accessing the same iSCSI volume from two different hosts at the same time as doing so may resuilt in corrupt data on the iSCSI volume.
+:::
+
+On the first node, check that the shared drive is mounted and note down the Drive letter.
+
+*![](/docs/images/clustered-listening-tentacles/shared-disk-properties.jpg)*
+
+Run through the Tentacle MSI Installer to install Tentacle Manager to its default location (C:\Program Files\Octopus Deploy\Tentacle). Do not click "get started" in the Tentacle manager, install the Octopus Tentacle instance using the command prompt as an Administrator by opening `cmd` then run these commands (replacing relevant values as appropriate):
+
+```batchfile
+cd "C:\Program Files\Octopus Deploy\Tentacle\"
+Tentacle.exe create-instance --instance "Tentacle" --config "C:\Octopus\Tentacle.config"
+Tentacle.exe configure --instance "Tentacle" --app "<sharedStorageDriveLetter>:\Octopus\Applications" --home "<sharedStorageDriveLetter>:\Octopus\Home" --port "10933" --noListen "False"
```

Comment:
Capitalisation is different between `<sharedStorageDriveLetter>` and `<OCTOPUS SERVER THUMBPRINT>`. Pick one style :)